### DISCUSSION TITLE
Any Update on adding Steam and Discord social profile icons in Profile settings?

### DISCUSSION THREAD

**Discussion by TankEngine-ish** [community/community#105238] [Other Feature Feedback, Questions, & Ideas]
I saw some threads that were talking about this throughout the years but it seems that Git Hub still haven't added them. I know it's a very minor thing but it would be nice to have a Steam icon below your profile bio on your profile.

**Comment by pjpscriv** [comment]
I created an extension as a workaround for this issue: https://chromewebstore.google.com/detail/github-social-icons/ledidlccnpfooopagmkgihgibfbnbbnp

Currently supports Steam links and will work towards adding Discord support too (I just need to find some examples of what discord profile links look like)

**Edit:** Discord links like `discord.gg/<USERNAME>` are now supported ✅
